packages/beacon-node/src/sync/unknownBlock.ts

```diff
@@ -651,14 +651,11 @@ export class BlockInputSync {
     // console.log("removeAndDownscoreAllDescendants", {block});
 
     for (const block of badPendingBlocks) {
-      //
-      // TODO(fulu): why is this commented out here?
-      //
-      //   this.knownBadBlocks.add(block.blockRootHex);
-      //   for (const peerIdStr of block.peerIdStrs) {
-      //     // TODO: Refactor peerRpcScores to work with peerIdStr only
-      //     this.network.reportPeer(peerIdStr, PeerAction.LowToleranceError, "BadBlockByRoot");
-      //   }
```

Comment:
as showed in #8465 when a block is orphaned, only some peers send this block, majority of peers cannot see it

so in this case we penalize innocent peers. That could be the reason why we commented it out?

I think we should not penalize peers, and note that issue in this code place instead